src/html.rs

```diff
@@ -33,10 +34,69 @@ enum TableState {
     Body,
 }
 
-struct HtmlWriter<'a, I, W>
-where
-    W: Write,
+/// Wrapper for String for which we can implement `StrWrite`.
```

Comment:
I see where these wrappers are coming from, the blanket impl for `Write` makes them necessary, but I wonder, what would it look like the other way, where `String` and `&mut impl StrWriter` impl the trait, but `Write` needs a wrapper?

Reply:
It's a good question, and one I will investigate!

Reply:
Your suggestion not only works well, it's also much cleaner! Thanks :-)